### ISSUE TITLE
SystemEntity and EntityManager

### ISSUE THREAD

**Issue by dlyr** [STORM-IRIT/Radium-Engine#767]
EntityManager create and manage the singleton SystemEntity.
This seems to be a design flaw since EntityManager can now be deleted.
As introduce in #763 , EntityManager dtor now deletes the SystemEntity, which seems odd for a singleton.
I propose to discuss this point in Discussions if needed, then to fix this issue.

**Comment by MathiasPaulin** [comment]
> EntityManager create and manage the singleton SystemEntity.
> This seems to be a design flaw since EntityManager can now be deleted.

 I agree. I opened a discussion (#768) to discuss how to solve this design flaw.
